### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,11 @@
...
+    if len(command.split()) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command.split()
+    if cp != "cp":
+        raise ValueError("Incorrect command.")
+    if file_in == file_out:
+        raise ValueError("Incorrect FileIn and FileOut.")
+    with open(file_in, "r") as f_in, open(file_out, "w") as f_out:
+        for line in f_in:
+            f_out.write(line)
```
Comment (Bohdan-Salamakha) [inline app/main.py:11]: u can write whole file, it's not necessary to write it line by line

File: app/main.py [hunk 2]
```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command.split()
+    if cp != "cp":
+        raise ValueError("Incorrect command.")
+    if file_in == file_out:
+        raise ValueError("Incorrect FileIn and FileOut.")
```
Comment (Bohdan-Salamakha) [inline app/main.py:8]: u don't need to do anything if files are the same
https://github.com/mate-academy/py-copy-file#:~:text=It%20must%20do%20nothing%20in%20case%20the%20user%20is%20trying%20to%20copy%20file%20to%20file%20with%20the%20same%20name.

File: app/main.py [hunk 3]
```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command.split()
```
Comment (Bohdan-Salamakha) [inline app/main.py:4]: don't call commad.split() twice
create a separated variable for it and call on demand

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
v1

File: app/main.py [hunk 4]
```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_sep = command.split()
+    if len(command_sep) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command_sep[0], command_sep[1], command_sep[2]
```
Comment (olena-hul) [inline app/main.py:5]:
```suggestion
    cp, file_in, file_out = command_sep
```

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**